### PR TITLE
Require libmambapy 1.5.3

### PR DESCRIPTION
**Pull request by jaimergp**
### Description

This [release](https://github.com/mamba-org/mamba/releases/tag/libmambapy-1.5.3) includes a few fixes that we have needed in conda-libmamba-solver for a while, including:

- Compatibility with `defaults::pkg` and `pkgs/main::pkg` specs (https://github.com/mamba-org/mamba/pull/2936)
- Ability to bypass the C++ level signal handling (https://github.com/mamba-org/mamba/pull/2946)

### Checklist - did you ...

- [x] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [x] Add / update necessary tests?
- [ ] Add / update outdated documentation?

### CODE REVIEW

**Comment by jaimergp** [comment]
Note that libmamba 1.5.3 is not on defaults yet, but the CI seems to be getting it from 1.5.3 from conda-forge anyway (even on the defaults CI entries), hence why everything's passing.

File: conda_libmamba_solver/mamba_utils.py [hunk 1]
```diff
@@ -88,25 +88,20 @@ def init_api_context() -> api.Context:
...
-    output_params = getattr(api_ctx, "output_params", api_ctx)
-    output_params.json = context.json
-    output_params.quiet = context.quiet
-    output_params.verbosity = context.verbosity
+    api_ctx.output_params.json = context.json
+    api_ctx.output_params.quiet = context.quiet
+    api_ctx.output_params.verbosity = context.verbosity
     api_ctx.set_verbosity(context.verbosity)
-    if output_params.json:
+    if api_ctx.output_params.json:
```
Comment (dholth) [inline conda_libmamba_solver/mamba_utils.py:95]: We are cancelling some other json if json is True?

**Review by kenodegard — APPROVED** [review]

File: conda_libmamba_solver/mamba_utils.py (hunk 1 above)
Comment (jaimergp) [reply conda_libmamba_solver/mamba_utils.py:95]: We don't want libmamba to print JSON on its own. It's something only conda should do.

**Review by jezdez — APPROVED** [review]

**Comment by jaimergp** [comment]
Merging with failures because they are connection errors.

**Pull request merged**